src/components/About.story.js

```diff
@@ -0,0 +1,78 @@
+import { storiesOf } from '@storybook/react';
+import overviewWrapper from '../storybook-addons/overviewWrapper';
+
+const stories = storiesOf('About', module);
+
+stories.add(
+    'Welcome',
+    overviewWrapper(`
+
+UI React is the Mudano component library.
+
+If you have questions ask the development team on Slack: [#ui-react](slack://channel?id=C89RXMYJY&team=T065RT3A7)
```

Comment:
only Mudano people can access this Slack channel, should we add this line?

I guess we also direct people to open issues in github (instead of Jira) for the same reason

Reply:
I'll remove Slack and direct towards GitHub issues :)